src/routes/router.ts

```diff
@@ -12,4 +14,7 @@ router.get('/user/:userId', getUser);
 router.post('/group', createGroup);
 router.get('/group/:groupId', getGroup);
 
+/* Venue Controller */
+router.get('/venue', getVenue);
+
```

Comment:
Realizing this is all old code... I deleted this file a week ago. Do a git pull to grab all the new changes. There is a functioning getVenues endpoint and you can alter that to reflect the comment above